db-client-java/src/main/java/com/eventstore/dbclient/EventStoreDBClient.java

```diff
@@ -9,93 +9,152 @@
 import java.util.List;
 import java.util.concurrent.CompletableFuture;
 
+/**
+ * Represents a client to a single node. A client instance maintains a full duplex communication to EventStoreDB.
```

Comment:
**SUGGESTION:** I think that `Represents a client to a single node` may be potentially confusing for the newbie as they may be concerned if it can connect to a cluster. I understand, that what you wrote below is true, but it is a bit too explicit imho.
```suggestion
 * Represents an EventStoreDB client. A client instance maintains a full-duplex communication to EventStoreDB. 
```